pkg/infrastructure/infinispan.go

```diff
@@ -22,6 +22,7 @@ import (
 	"k8s.io/api/apps/v1"
```

Comment:
why not group infinispan and other infra on the same package?
pkg/infinispan/{controller,infra}? 
does it make sense?

Reply:
Thanks for pointing it out. There are lots of pck structure changes needs to be done so we have created seperate Jira for it(KOGITO-3322). We could incorporate your suggested changes with that Jira